### DISCUSSION TITLE
v3.8.2_z: version 12.85

### DISCUSSION THREAD

**Discussion by Zbizu** [Zbizu/rme#5] [Announcements]
## Changes
- added 12.85
- official otb v3.61 (12.85) (credits: kokokoko)

![rme382](https://user-images.githubusercontent.com/5205079/148659495-823dbb89-40e2-46d0-a007-5e14381580ae.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Zbizu/rme/releases/tag/v3.8.2_z'>v3.8.2_z: version 12.85</a>.</em>

**Comment by Elveniane** [comment]
A good idea to improve your version, would be to add the option to paste positions XYZ into portals. You can currently copy the xyz position, but it cannot be pasted in any way when creating a teleports. Could use such a button in the properties of teleports. ;)
